README.md

```diff
@@ -13,6 +13,32 @@ This is the R package for simulating spread of pests and pathogens. The package
+
+* Susceptible-infected (`SI`) and susceptible-exposed-infected (`SEI`) model types (`model_type`, `latency_period`).
+* Host mortality tracking (`mortality_rate`, `mortality_time_lag`, `mortality`).
+* Host removal and pesticide application treatments (`treatments`, `treatment_date`, `pesticide_duration`).
+* Host resistance based on pesticide application treatments (`pesticide_duration` > 0).
+* Treatments applied only to a ratio of hosts (`treatment_application`).
+* Yearly pest removal based on lethal temperature (`lethal_temperature`, `lethal_month`).
+* Two different dispersal kernels (`natural_dispersal_kernel`, `anthropogenic_dispersal_kernel`).
+* Cauchy, Exponential, Uniform, Power-law, Deterministic neighbor, Hyperbolic-Secant, Gamma, Weibull, Normal, and Logistic radial dispersal kernels use Von Mises distribution.
+* Seasonal spread (`seasonality` in months).
+* Host movement for animals moving from farm to farm or plants via nursery trade (`use_movements`, `movements_file`).
+* Reduced stochasticity options (`generate_stochasticity`, `establishment_stochasticity`, `movement_stochasticity`) and deterministic versions of kernels (`deterministic`) other required parameters if reducing stochasticity are (`dispersal_percentage`, `establishment_probability`).
+* Spread rate measurement in 4 cardinal directions (`west_rate`, `east_rate`, `south_rate`, `north_rate`) when (`use_spreadrates`) is true.
+* Distance to quarantine in 4 cardinal directions (`north_distance_to_quarantine`, `south_distance_to_quarantine`, `east_distance_to_quarantine`, `west_distance_to_quarantine`) when (`use_quarantine`) is true and (`quarantine_areas_file`) is provided.
+* Probability of quarantine escape (`escape_probability`).
+* Overpopulation function (individuals in areas of high population leave the area and disperse longer distances on average) (`use_overpopulation`, `overpopulation_percentage`, `leaving_percentage`, `leaving_scale_coefficient`).
+* Flexible output frequency with n number of days, weeks, months, or years as options (`output_frequency`, `output_frequency_n`).
+
+### Functions in rpops
+* `calibrate:` Calibration of the model parameters using either MCMC (markov chain monte carlo) or ABC (approximate bayesian computation). 
+* `validate:` Validation of the model using quantiy, allocation, and configuration disagreement.
+* `pops_multirun:` Parallel execution of multiple stochastic runs (`number_of_cores` used to set cores used if left NULL defaults to using n - 1 cores on the machine). Outputs statistics of infected/infested hosts across multiple runs (`simulation_mean`,  `single_run`,  `simulation_sd`, `simulation_min`, `simulation_max`), current state using the median (`infected`,  `exposed`, and `susceptible`), Average and standard deviations whole area statistics (`number_infecteds`, `infected_areas`), and probabilty of infection (`probability`) which is the percent of model runs that a cell has at least one infestation/infection.
```

Comment:
There was some mixup, the changes got lost:

```suggestion
* `pops_multirun:` Parallel execution of multiple stochastic runs (`number_of_cores` used to set cores used if left NULL defaults to using n - 1 cores on the machine). Outputs statistics of infected/infested hosts across multiple runs (`simulation_mean`,  `single_run`,  `simulation_sd`, `simulation_min`, `simulation_max`), current state using the median (`infected`,  `exposed`, and `susceptible`), average and standard deviations whole area statistics (`number_infecteds`, `infected_areas`), and probability of infection (`probability`) which is the percent of model runs that a cell has at least one infestation/infection.
```

I assume `number_infecteds` is intentional since it is in the source code.